source/nvwave.py

```diff
@@ -165,6 +169,8 @@ def open(self):
 		"""
 		with self._waveout_lock:
 			if self._waveout:
+				if self._closeTimer:
+					wx.CallAfter(self._closeTimer.Stop)

```

Comment:
Don't you somehow need to set self._closeTimer to None hereafter? May be I'm just missing something.